.dockerignore

```diff
@@ -0,0 +1,15 @@
+.github
+.pytest_cache
+.venv
+.env
+.env.dev
+.env.template
+.gitignore
+env_vars.yml
+Inventory
+Playbook
+poetry.lock
```

Comment:
If we switch to Poetry (as suggested below) then we _will_ want this `poetry.lock` file available from within our container